www/class/centreonWidget/Params/Connector/Poller.class.php

```diff
@@ -89,6 +89,7 @@ public function getListValues($paramId)
                     );
                 }
 
+                $tab = array(null => null);
```

Comment:
```suggestion
                $tab = [];
```
Could you check with an empty array if this correct the bug ?